docs/book/requirements.rst

```diff
@@ -0,0 +1,82 @@
+.. index::
+    single: System Requirements
+
+System Requirements
+===================
+
+Here you will find the list of system requirements that have to be adhered to be able to use **Sylius**.
+First of all have a look at the `requirements for running Symfony <http://symfony.com/doc/current/reference/requirements.html>`_.
+
+Read about the `LAMP stack <https://en.wikipedia.org/wiki/LAMP_(software_bundle)>`_ and the `MAMP stack <https://en.wikipedia.org/wiki/MAMP>`_.
+
+Operating Systems
+-----------------
+
+The recommended operating systems for running Sylius are the Unix systems - **Linux, MacOS**.
+
+Web server and configuration
+----------------------------
+
+We do recommend using Apache web server ≥ 2.2.
+Remember to load the ``php`` module in the ``/etc/apache2/httpd.conf``.
+
+PHP required modules and configuration
+--------------------------------------
+
+**PHP version**:
+
++---------------+-----------------------+
+| PHP           | ^5.6|^7.0             |
++---------------+-----------------------+
+
+**PHP extensions**:
+
++---------------+---------------------------+
+| ext-gd        | No specific configuration |
++---------------+---------------------------+
+| ext-exif      | No specific configuration |
++---------------+---------------------------+
+| ext-fileinfo  | No specific configuration |
++---------------+---------------------------+
+| php5-intl     | No specific configuration |
++---------------+---------------------------+
+
+**PHP configuration settings**:
+
++---------------+-----------------------+
+| memory_limit  | ≥512M                 |
++---------------+-----------------------+
+| date.timezone | Europe/Warsaw         |
++---------------+-----------------------+
+
+.. warning::
+
+    Use your local timezone, for example America/Los_Angeles or Europe/Berlin. See http://www.php.net/timezones for the list of all available timezones.
+
+.. note::
+
+    While using PHP 7.0 the memory limit may be decreased to 256M.
```

Comment:
I have fail this project... 👎 #oliver So, we should define minimal memory to 1024 MB then